frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -412,6 +417,14 @@ yarn publish dist/<pkg> --no-git-tag-version --new-version <version>
 
 If the given package doesn't exist in npm registry, add `--access public` to `yarn publish` command.
 
+If the newly published version is lesser than the latest published version in terms of semver rules
```

Comment:
```suggestion
If the newly published version is earlier than the latest published version in terms of semver rules
```
is semver correct? If it is then I think we should put it in backticks.

Reply:
@opayne1 Hi, this is a reference to https://semver.org/ specification.

For example, version `1.0.0` is "lesser" or "lower" than version `1.1.0`. I didn't use the quotes above when writing the word lesser, should I put them in?

Using the word "earlier" (without quotes) is less accurate in this context, in my opinion.

Reply:
Can we say "Semantic Versioning" instead. Just for clarity? 
Also, we always us "earlier" to refer to a previous software version or fix level in the documentation, so to stay consistent with docs we should use earlier here. Technically using lesser in this sentence is grammatically incorrect.

Reply:
Also, you dont need quotes around earlier

Reply:
Okay I am good with semver then. And lets go with comes before if we can't go with earlier. I see what you are saying though. I think comes before does read better than lesser imo.